.github/workflows/deployment.yaml

```diff
@@ -28,102 +28,98 @@ jobs:
         run: |
 
           cp template.env .env
-          
+
         working-directory: ./
 
       - name: Echo .env
         run: cat .env
         working-directory: ./
-      
 
       - name: Wait for 20 seconds
         run: sleep 10
-      
+
       - name: Check Database Health
-        run: nc -zvw 1 cozinha_na_cozinha_mysql 3306
+        run: nc -zvw 1 data_labdsoft 3306
         continue-on-error: true
       # Setup java version
       - name: Set up Java 17
         uses: actions/setup-java@v4
         with:
-          distribution: 'temurin'
-          java-version: '17'
-          cache: 'maven'
-          cache-dependency-path: './pom.xml'
-        
+          distribution: "temurin"
+          java-version: "17"
+          cache: "maven"
+          cache-dependency-path: "./pom.xml"
+
       - name: Maven Clean
         run: mvn clean
         working-directory: ./
 
-      - name: Maven Test
-        run: mvn test
```

Comment:
Why is not running tests?

Reply:
Passou a rodar com o script do go que está um pouco mais a baixo ou a cima, se os testes falharem ele não cria o jar e a pipeline falha

Reply:
Falha na build?

Reply:
sim falha se não houver o .jar